src/gt4py/next/program_processors/runners/dace_fieldview/transformations/auto_optimize.py

```diff
@@ -90,6 +91,8 @@ def gt_auto_optimize(
             one for all.
         blocking_dim: On which dimension blocking should be applied.
         blocking_size: How many elements each block should process.
+        require_independent_nodes: If `True` only apply loop blocking if there
```

Comment:
```suggestion
        blocking_only_if_independent_nodes: If `True` only apply loop blocking in a dimension X if the map contains nodes that do not depend on X.
```

Reply:
The whole documentation and naming uses the word dependent and independent so we should stuck to it too.
I updated the description of the `LoopBlocking` transformation to be a bit more clearer.
I also added a reference to the the `LoopBlocking` transformation for that parameter.